packages/core/saved-objects/core-saved-objects-common/src/types.ts

```diff
@@ -17,3 +17,22 @@ export interface SavedObjectTypeIdTuple {
   /** The type of the saved object */
   type: string;
 }
+
+// NOTE: moved from x-pack/plugins/spaces/common/types.ts for use by SO security ext
+/**
+ * Client interface for interacting with legacy URL aliases.
+ */
+export interface LegacyUrlAliasTarget {
```

Comment:
Why is this in the `common` package?

Reply:
It is referenced in the following locations:
```
- packages/core/saved-objects/core-saved-objects-server/src/extensions/security.ts
- x-pack/plugins/spaces/public/share_saved_objects_to_space/components/types.ts
- x-pack/plugins/security/server/saved_objects/saved_objects_security_extension.ts
- x-pack/plugins/security/server/saved_objects/saved_objects_security_extension.test.ts
- x-pack/plugins/spaces/public/spaces_manager/spaces_manager.ts
- x-pack/plugins/spaces/server/spaces_client/spaces_client.ts
```

I moved it from `x-pack/plugins/spaces/common/types.ts` so I could reference it in the security extension definition in packages. I could instead make a redundant/compatible server package definition and keep the original definition with the plugin. Would this be preferable?

Reply:
I wasn't sure it was used by browser-side code. But you confirmed it is, so it's fine to have it in the common package, thanks!